telemetry/telemetry.go

```diff
@@ -49,7 +49,7 @@ var (
 func Configure(configuration *Configuration) (*Telemetry, error) {
 	return &Telemetry{
 		Metrics: &Metrics{
-			Meter:         otel.Meter("openfga-sdk/0.6.0"),
+			Meter:         otel.Meter("openfga-sdk/0.6.1"),
```

Comment:
I just noticed this differs from `defaultUserAgent`, is taht expected?

Reply:
The user agents are consistent across the SDKs, but the metrics name is unrelated and should not contain the user agent or the version.

`openfga-sdk` would work for that, and yes let's make the metric name across all the SDKs

Reply:
@rhamzeh I believe we're currently using the `version` in the meter name across all SDKs. We can update it, but they'll need updated.

There are inconsistencies in some of the SDKs' agent strings. Python, for example, reports the `User-Agent` as "openfga-sdk (python) X.Y.Z" in its OAuth2 class, although it is reported as "openfga-sdk python/X.Y.Z" in the API classes. I haven't checked all of them to see if this is the case everywhere.